test/cases/fn_typeinfo_passed_to_comptim_fn.zig

```diff
@@ -0,0 +1,17 @@
+const std = @import("std");
```

Comment:
I think there might be a small typo on the filename here as well. Did you mean `fn_typeinfo_passed_to_comptime_fn.zig`?